R/elasticsearch_parsers.R

```diff
@@ -391,7 +391,8 @@ unpack_nested_data <- function(chomped_df, col_to_unpack) {
         futile.logger::flog.fatal(msg)
         stop(msg)
     }
-    if (!("character" %in% class(col_to_unpack)) || length(col_to_unpack) != 1) {
+    if (!("character" %in% class(col_to_unpack)) || length(col_to_unpack) != 
+        1) {
```

Comment:
please put this `1) {` back on the line above, so the line ends with the opening `{`